docs/overview.md

```diff
@@ -49,3 +49,24 @@ To learn how to use popular third-party libraries that connect with RDFLib, plea
 
 pyjelly includes its own [generic API](generic-sink.md) for working with RDF data (no RDFLib or other external RDF libraries required).  
 It provides simple, built-in types for triples and quads, allowing you to create, read, and write data directly in the Jelly format.
+
+## Accessing stream frame metadata 
```

Comment:
Please move this to the end of the generic API page, this is very much advanced usage.